### PR TITLE
OBPIH-3593 Improve product sources page

### PR DESCRIPTION
**Pull request by DominikaHatala**

### CODE REVIEW

**Review by awalkowiak — APPROVED** [review]

File: grails-app/controllers/org/pih/warehouse/product/ProductSupplierController.groovy [hunk 1]
```diff
@@ -27,7 +27,29 @@ class ProductSupplierController {
...
+            if (params.supplierId) {
+                eq('supplier.id', params.supplierId)
+            }
+            if (params.manufacturerId) {
+                eq('manufacturer.id', params.manufacturerId)
+            }
+            if (params.q) {
+                or {
+                    ilike("productCode", "%" + params.q + "%")
+                    ilike("code", "%" + params.q + "%")
```
Comment (jmiranda) [inline grails-app/controllers/org/pih/warehouse/product/ProductSupplierController.groovy:43]: We probably need to match against Product properties too but I'll leave that up to Kelsey. See my comments in the ticket.

File: grails-app/views/taglib/_autoSuggest.gsp [hunk 2]
```diff
@@ -3,7 +3,7 @@
 <input id="${attrs.id}-value" class="value" type="hidden" name="${attrs.name}.value" value="${attrs.valueId}" ${attrs.valueDataBind }/>
 <input id="${attrs.id}-suggest" type="text" ${attrs.textDataBind}
        class="autocomplete text ${attrs.styleClass}" name="${attrs.name}-name" placeholder="${attrs.placeholder}" value="${attrs.valueName}"
-       style="width: ${attrs.width}px; display: ${attrs.suggestDisplay};" size="${attrs.size}">
+       style="width: ${attrs.width}; display: ${attrs.suggestDisplay};" size="${attrs.size}">
```
Comment (jmiranda) [inline grails-app/views/taglib/_autoSuggest.gsp:6]: This seems like a reasonable refactoring. Just make sure it doesn't break existing autoSuggest fields.

**Comment by jmiranda** [comment]
LGTM but I want to hear from Kesley re: the concerns I raised in the ticket. If she doesn't care, go ahead and merge it.

**Comment by jmiranda** [comment]
@DominikaHatala I'm going to merge this, but please check with Kelsey about the comments I made.

**Pull request merged**